pulp_smash/api.py

```diff
@@ -192,8 +203,9 @@ class Client(object):
     >>> response = client.get('/index.html')  # Do default SSL verification
 
     Anything accepted by the `Requests`_ functions may be placed in
-    ``request_kwargs`` or passed in to a specific call. You can set ``verify``,
-    ``auth`` and more.
+    ``request_kwargs`` or passed in to a specific call. You can set ``verify``
+    for example. ``scheme``, on the other hand, will be removed since it is
+    there only to tell if the system should be accessed over HTTP or HTTPS.
```

Comment:
If one writes `client.request_kwargs['scheme'] = ...`, then `client.request_kwargs['scheme']` will be updated as expected. Similarly, if one writes `client.get('/some/path', scheme='http')`, then that client's `request_kwargs` will be updated before the HTTP GET request is issued. In this context, I don't think it's accurate to say that `scheme, on the other hand, will be removed.`.